manifest.toml

```diff
@@ -60,6 +60,13 @@ ram.runtime = "512M"
     type = "boolean"
     default = true
 
+    [install.version_to_follow]
+    ask.en = "Upgrade channel to use"
+    help.en = "Recent major releases contains sometimes some annoying bugs or regressions. Using the oldstable channel during this period can be useful if you need high stability."
+    type = "select"
+    choices = ["stable", "oldstable"]
```

Comment:
I find those two choices confusing... Is Oldstable defined by Nextcloud? shouldn't we need to be more specific about it